pkg/resources/pods_test.go

```diff
@@ -72,6 +72,37 @@ func TestPodsSortedByAge(t *testing.T) {
 			pod("enter-sandman", withStartTime(time.Now()), withIP("1.2.3.5")),
 		},
 		want: []string{"2.3.4.5", "1.2.3.4", "1.2.3.5", "3.4.5.6"},
+	}, {
+		name: "one pod, but can't use",
+		pods: []*corev1.Pod{
+			pod("unforgiven", withStartTime(aTime), withIP("1.1.1.1"), withPhase(corev1.PodPending)),
+		},
+		want: []string{},
+	}, {
+		name: "one pod, but can't use II",
+		pods: []*corev1.Pod{
+			pod("unforgiven-ii", withStartTime(aTime), withIP("1.1.1.1"), withPhase(corev1.PodPending),
```

Comment:
This pod will be filtered out due to `PodPending`. Is `DeletionTimestamp` necessary here? Shall we change the pod phase to `PodRunning`?

Reply:
🤦 
I thought I changed that.
Pushed.